jsonnet/dashboards/apps/ssl-exporter.libsonnet

```diff
@@ -0,0 +1,253 @@
+/*
+  Copyright 2020 The dNation Kubernetes Monitoring Authors. All Rights Reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
```

Comment:
Just a cosmetic change to be consistent with other dashboards. Please add space below and add name of dashboard like `/* SSL exporter dashboard */` or something like that.